triggering-jenkins.md

```diff
@@ -0,0 +1,93 @@
+# Triggering a Jenkins build as a task on XebiaLabs
+
+This guide will demonstrate how to set a XebiaLabs task to trigger a Jenkins build. 
+
+## Prerequisites
+
+* Jenkins is installed
+
+* Have an account with XebiaLabs
+
+#### 1.1 Create a new Jenkins job
+
+On the left side of the site, click on 'New Item' to create a new project. 
+
+![new-item](assets/new-item.png)
+
+Enter in a project name and select the project type 'Freestyle project'. 
+
+![new-project](assets/new-project.png)
+
+In the configuration settings, go the 'Build' section. Click on 'Add build step' and select 'Execute shell'. 
+
+![build-step](assets/build-step.png)
+
+Enter the necessary commands. The below example will execute the android-app-test script sample from Kobiton.
+
+```
+#!/bin/bash
+[ ! -d "samples" ] && git clone https://github.com/kobiton/samples.git || echo "Samples already exist"
+cd samples/javascript
+npm install
+npm run android-app-test
+```
+
+In your automation test, make sure to include your own Kobiton username and API key as well as the desired device and platform you wish to test with, which can be found on the Kobiton website.
+
+This guide is executing the NodeJS test script from the Kobiton samples, so we will add the NodeJS Plugin on Jenkins to execute NodeJS script as a build step. To do this, go to 'Manage Jenkins' -> 'Manage Plugins' -> Available. Search for NodeJS and install it.
+
+![manage-plugins](assets/manage-plugins.png)
+
+If you are using a different language, you may try to find a suitable plugin for your project if necessary.
+
+Under build environment, check the box that says 'Provide Node & npm bin/ folder to PATH'.
+
+![node-path](assets/node-path.png)
+
+Save your configuration settings. Try 'Build Now' and then check Kobiton cloud devices to see if a test session was created. If successful, you can move onto triggering a Jenkins build from a XebiaLabs task. 
```

Comment:
"you can move onto triggering a Jenkins build from a XebiaLabs task" Is it related to the `Create a new Jenkins job` ???